tests/integration/test_snap.py

```diff
@@ -167,6 +167,12 @@ def test_hold_refresh():
     assert f"hold: {hold_date}" in result.decode()
 
 
+def test_forever_hold_refresh():
+    snap.hold_refresh(forever=True)
+    result = check_output(["snap", "refresh", "--time"])
+    assert "hold: 2315-06-30" in result.decode()
```

Comment:
What's this magical date? Will it change as time progresses, or is it a fixed date? On my system it's "2315-07-01", maybe due to me being on NZDT, or because it's already tomorrow for me? :-) If it is a fixed date, I recommend we do `assert "hold: 2315" in result.decode()`, if not, perhaps we just check with `snap get system refresh.hold`?